src/check_config_provider/redis_config_provider.rs

```diff
@@ -339,6 +338,28 @@ pub fn run_config_provider(
     })
 }
 
+pub fn determine_owned_partitions(config: &Config) -> HashSet<u16> {
+    // Determines which partitions this checker owns based on number of partitions,
+    // number of checkers and checker number
+    let checker_number: u16 = config
+        .checker_id
+        .split('-')
+        .last()
+        .expect("checker_id should be in format <name>-<checker_number>")
+        .parse()
+        .expect("checker_id does not contain a valid checker_number");
```

Comment:
```suggestion
        .expect("checker_id should contain a valid checker_number");
```

keeps with the "expect <sentence that follows expect>"